tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/typedefs/TypeDefinitionsTest.java

```diff
@@ -169,19 +169,19 @@ public void testBinaryExprAssignments() {
         BRunUtil.invoke(compileResult, "testBinaryExprAssignments");
     }
 
-//    @Test
-//    public void testRecordTypeResolving() {
-//        JvmRunUtil.invoke(recordFieldRes, "testRecordTypeResolving");
-//    }
-//
-//    @Test
-//    public void testRecordTypeResolvingWithTypeInclusion() {
-//        JvmRunUtil.invoke(recordFieldRes, "testRecordTypeResolvingWithTypeInclusion");
-//    }
+    @Test
+    public void testRecordTypeResolving() {
```

Comment:
Shall we include a test, where we include a record type from a different module?

Reply:
Done